.eslintrc.json

```diff
@@ -67,7 +67,6 @@
         "no-empty-function": "off",
         "@typescript-eslint/no-empty-function": "error",
         "@typescript-eslint/no-extra-non-null-assertion": "error",
-        "@typescript-eslint/no-floating-promises": "error",
         "@typescript-eslint/no-for-in-array": "error",
```

Comment:
Keep the removed rule. Linting is for improving code quality by enforcing strict rules, so better fix issues instead of removing rules.